packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -488,22 +463,16 @@ impl EstimablePredicates for ScriptTransaction {
     }
 }
 
+impl GasValidation for CreateTransaction {}
```

Comment:
I'd remove the default trait method impl and move the logic here.

It seems a clearer way to say `CreateTransaction` has no opinions on gas validity.